string command
uint8[]  ids
float64[]  goal_angles